src/renderer/components/SideBar/SideBarListItem.js

```diff
@@ -11,6 +11,7 @@ export type Props = {
   label: string | (Props => React$Node),
   desc?: Props => any, // TODO: type should be more precise, but, eh ¯\_(ツ)_/¯
   icon?: any, // TODO: type should be more precise, but, eh ¯\_(ツ)_/¯
+  iconSize?: number;
```

Comment:
Lint error on the `;` (should be `,`)

Reply:
ah good eyes thanks!

Reply:
my config is so bad that for me everything in Flow is red anyway haha, hard to catch lint errors like this
<img width="238" alt="Screenshot 2022-03-02 at 17 14 50" src="https://user-images.githubusercontent.com/91890529/156402020-da74e526-d787-4dee-a83d-63163f2a26b9.png">